server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -37,7 +37,8 @@ const createHttpError = require('http-errors');
 const HttpServerInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/HttpServerInterface');
 const OperationClientInterface = require('onf-core-model-ap/applicationPattern/onfModel/models/layerProtocols/OperationClientInterface');
```

Comment:
We can see OperationClientInterface also has a duplicate import statement.  Can we remove this and change corresponding lines of OperationClientInterface to operationClientInterface.

Reply:
Thanks, removed the duplicate entry.